### PR TITLE
sys/shell: fix cppcheck warnings

### PR DESCRIPTION
**Pull request by makomi**
Inspired by #480, I used `cppcheck` to cleanup some things. This commit is part of a series of commits (s.a. #5764).

### CODE REVIEW

File: sys/shell/commands/sc_ccnl.c [hunk 1]
```diff
@@ -148,11 +148,10 @@ int _ccnl_content(int argc, char **argv)
 static struct ccnl_face_s *_intern_face_get(char *addr_str)
 {
     /* initialize address with 0xFF for broadcast */
-    size_t addr_len = MAX_ADDR_LEN;
     uint8_t relay_addr[MAX_ADDR_LEN];
     memset(relay_addr, UINT8_MAX, MAX_ADDR_LEN);
+    size_t addr_len = gnrc_netif_addr_from_str(relay_addr, sizeof(relay_addr), addr_str);;
```
Comment (aabadie) [inline sys/shell/commands/sc_ccnl.c]: harmless but it seems you added an extra `;` at the end of this line.

**Comment by aabadie** [comment]
ACK when commits are squashed. Just keep one with comment: "sys/shell: fix cppchek warnings"

**Comment by makomi** [comment]
Hm, I can't find the static tests that failed and I don't know why the semi-colon would make a difference. Any suggestions?

**Comment by miri64** [comment]
@makomi the tests fail because the "NEEDS SQUASHING" label is still set. Please squash the commits into one (as suggested by @aabadie) so we can remove it.

**Pull request merged**